lib/email/receiver.rb

```diff
@@ -861,6 +872,45 @@ def process_forwarded_email(destination, user)
       true
     end
 
+    def forwarded_email_quote_forwarded(destination, user)
+      embedded = embedded_email_raw
+      raw = <<~EOF
+        #{@before_embedded}
+
+        [quote="#{I18n.t('emails.forwarded_message')}"]
+        #{PlainTextToMarkdown.new(embedded).to_markdown}
+        [/quote]
+        EOF
+      title = subject
+
+      case destination[:type]
```

Comment:
Any way we could avoid duplicated that logic between `forwarded_email_create_replies` and `forwarded_email_quote_forwarded`?

Reply:
Yep, I had to use a block for creating the embedded user at the correct point in the logic, but it seems pretty clean to me.